cypress/e2e/Testing/client-user.cy.js

```diff
@@ -0,0 +1,64 @@
+describe("User tests", () => {
+    it("CAN login and view their profile", () => {
+      let email = "Norofftest@noroff.no";
+      let password = "Norofftest@noroff.no";
+  
+      cy.visit("https://noroff-anders.github.io/Workflow-CA/");
+      cy.title().should("contain", "Test Client");
+      cy.wait(1000);
+  
+      cy.contains("Login").click({ force: true });
+      cy.wait(1000);
+  
+
+      cy.get("#loginEmail").type(email);
+      cy.get("#loginPassword").type(password);
+      cy.wait(1000);
+      cy.get("button.btn.btn-success").eq(1).click({force: true});
+      cy.wait(1000);
+    });
+
+
+    it("CAN use logout button to logout", () => {
+        let email = "Norofftest@noroff.no";
+        let password = "Norofftest@noroff.no";
+    
+        cy.visit("https://noroff-anders.github.io/Workflow-CA/");
+        cy.title().should("contain", "Test Client");
+        cy.wait(1000);
+    
+        cy.contains("Login").click({ force: true });
+        cy.wait(1000);
+    
+  
+        cy.get("#loginEmail").type(email);
+        cy.get("#loginPassword").type(password);
+        cy.wait(1000);
+        cy.get("button.btn.btn-success").eq(1).click({force: true});
+        cy.wait(4000);
+
+        cy.contains("Logout").click({ force: true });
+        cy.wait(1000);
```

Comment:
Same thing here, how do you prove it?

Reply:
Added: 
``` cy.get("button[data-visible=loggedOut").should("exist");```
After logout  this checks if data-visible=loggedOut exists